### PR TITLE
cbonsai: init at 1.0.0

### PR DESCRIPTION
**Pull request by manveru**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 115062` at 78afd084 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/fe7ef9d02a0c08e28d09ca9e2f8d5a98">cbonsai</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/games/cbonsai/default.nix:18:5:
```
   |
18 |     license     = with licenses; [ gpl3 ];
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/games/cbonsai/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  installFlags = [ "PREFIX=$(out)" ];
+
+  meta = with lib; {
+    description = "grow bonsai trees in your terminal";
+    homepage    = "https://gitlab.com/jallbrit/cbonsai";
+    license     = with licenses; [ gpl3 ];
+    # Most of abuse is free (public domain, GPL2+, WTFPL), however the creator
+    # of its sfx and music only gave Debian permission to redistribute the
+    # files. Our friends from Debian thought about it some more:
+    # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=648272
```
Comment (ninjin) [inline pkgs/games/cbonsai/default.nix:22]: Hmm… Seems unrelated. Copy paste error?

Comment (manveru) [reply pkgs/games/cbonsai/default.nix:22]: you're right

File: nixos/tests/bat.nix [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import ./make-test-python.nix ({ pkgs, ... }: {
+import ./make-test-python.nix ({ pkgs, ... }:
```
Comment (SuperSandro2000) [inline nixos/tests/bat.nix:1]: Can you revert this unrelated change?

File: pkgs/games/cbonsai/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, fetchFromGitLab, ncurses, pkg-config }:
+stdenv.mkDerivation rec {
+  version = "1.0.0";
+  pname = "cbonsai";
+  src = fetchFromGitLab {
```
Comment (SuperSandro2000) [inline pkgs/games/cbonsai/default.nix:5]:
```suggestion
{ stdenv, lib, fetchFromGitLab, ncurses, pkg-config }:

stdenv.mkDerivation rec {
  pname = "cbonsai";
  version = "1.0.0";

  src = fetchFromGitLab {
```

File: pkgs/games/cbonsai/default.nix [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    owner = "jallbrit";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1jc34j627pnyjgs8hjxqaa89j24gyf0rq9w61mkhgg0kria62as7";
+  };
+  nativeBuildInputs = [ pkg-config ];
```
Comment (SuperSandro2000) [inline pkgs/games/cbonsai/default.nix:11]:
```suggestion
  };

  nativeBuildInputs = [ pkg-config ];
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: nixos/tests/bat.nix (hunk 2 above)
Comment (manveru) [reply nixos/tests/bat.nix]: Not sure how I missed that, thanks.

File: pkgs/games/cbonsai/default.nix (hunk 4 above)
Comment (manveru) [reply pkgs/games/cbonsai/default.nix]: done :)

File: pkgs/games/cbonsai/default.nix (hunk 3 above)
Comment (manveru) [reply pkgs/games/cbonsai/default.nix]: done :)

**Comment by manveru** [comment]
Thanks for the review, should look a bit nicer now.

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 115062` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cbonsai</li>
  </ul>
</details>

**Review by siraben — APPROVED** [review]
LGTM. Pretty!

File: pkgs/games/cbonsai/default.nix [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ ncurses ];
+  installFlags = [ "PREFIX=$(out)" ];
+
+  passthru.updateScript = nix-update-script { attrPath = pname; };
+
+  meta = with lib; {
+    description = "grow bonsai trees in your terminal";
```
Comment (jonringer) [inline pkgs/games/cbonsai/default.nix:21]:
```suggestion
    description = "Grow bonsai trees in your terminal";
```

**Review by jonringer — APPROVED** [review]
otherwise LGTM :)
